src/Nethermind/Nethermind.Merge.Plugin/Handlers/EngineRpcCapabilitiesProvider.cs

```diff
@@ -30,6 +30,7 @@ public EngineRpcCapabilitiesProvider(ISpecProvider specProvider)
             _capabilities[nameof(IEngineRpcModule.engine_forkchoiceUpdatedV1)] = (true, true);
             _capabilities[nameof(IEngineRpcModule.engine_getPayloadV1)] = (true, true);
             _capabilities[nameof(IEngineRpcModule.engine_newPayloadV1)] = (true, true);
+            _capabilities[nameof(IEngineRpcModule.engine_getClientVersionV1)] = (true, true);
```

Comment:
```suggestion
            _capabilities[nameof(IEngineRpcModule.engine_getClientVersionV1)] = (true, false);
```
Second value should be false as we dont need a warning if it is missing when exchanging capabilities with CL